src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -54,178 +69,171 @@ class SkillRatingCard extends Component {
   };
 
   render() {
-    let ratingMessage = '';
-
-    if (this.props.rating === 0) {
-      ratingMessage = 'The skill has not been rated by you';
-    } else if (this.props.rating === 1) {
-      ratingMessage = 'Hated it';
-    } else if (this.props.rating === 2) {
-      ratingMessage = 'Disliked it';
-    } else if (this.props.rating === 3) {
-      ratingMessage = "It's OK";
-    } else if (this.props.rating === 4) {
-      ratingMessage = 'Liked it';
-    } else if (this.props.rating === 5) {
-      ratingMessage = 'Loved it';
+    const { chartWidth, ratingsOverTimeWidth, offset } = this.state;
+    let ratingMessage;
```

Comment:
Use ES6 destructuring for props. `this.props` has to be written multiple times otherwise.

Reply:
Yes